fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -1470,6 +1470,86 @@ def _free_full_params(self, params: Optional[List[Parameter]] = None) -> None:
             # the Storage to 0 to save memory.
             free_storage_(p._full_param_padded)
 
+    def local_metadata_dict(self) -> Dict[str, list]:
+        """
+        Get the information needed to reconstruct the model from shards offline.
+        """
+
+        def clean_path(path: str):
```

Comment:
move this to the end of the file and rename to _clean_path? this seems like something other functions can use in the future as well.